www/src/pages/rect.mdx

```diff
@@ -42,36 +42,32 @@ Hook that observes and returns the measurements (ClientRect) of a DOM element. P
 If `observe` is false, the element's rect will no longer be observed, pass it `true` again and it will. This is mostly used for things like popovers and animations, so you can usually observe when you need to, and stop observing when you don't.
 
 ```.jsx
-(() => {
```

Comment:
Why remove the iife?

Reply:
When there’s a single component in a code block, it will call it automatically. In my head, keeping the iife was just unneccesary syntax noise.

Reply:
I asked @ryanflorence about it and he said the reason he originally used the iife's was because react-live needs a component instance to be returned, not the component class itself. Is that not the case here?

Reply:
Hm okay, I don’t think that’s the case anymore at least. I’m on my phone right noe, but the gif on their readme looks like a regular function. Perhaps they invoke it for you? 🤷‍♂️ I’ll verify tomorrow morning 🙌

Reply:
Yep it's not needed anymore. You can check it yourself by going to https://ui.reach.tech/rect/ and removing the iife if you want :)